winter/http/throttling.py

```diff
@@ -26,6 +27,8 @@ class Throttling:
 
 
 def throttling(rate: typing.Optional[str], scope: typing.Optional[str] = None):
+    if scope is None:
```

Comment:
it was mistake
without it, each request with thottling has same thottling

Reply:
Can't find a test which checks it.

Reply:
Added